src/workflows-app/SubmissionDetails.js

```diff
@@ -61,20 +70,24 @@ export const BaseSubmissionDetails = ({ name, namespace, workspace, submissionId
   const loadRuns = useCallback(
     async (cbasUrlRoot) => {
       try {
-        const runsResponse = await Ajax(signal).Cbas.runs.get(cbasUrlRoot, submissionId);
-        const runsAnnotatedWithDurations = _.map(
-          (r) => _.merge(r, { duration: getDuration(r.state, r.submission_date, r.last_modified_timestamp, isRunInTerminalState) }),
-          runsResponse.runs
-        );
-        setRunsData(runsAnnotatedWithDurations);
-        setRunsFullyUpdated(runsResponse.fully_updated);
+        return await Ajax(signal).Cbas.runs.get(cbasUrlRoot, submissionId);
       } catch (error) {
         notify('error', 'Error loading saved workflows', { detail: error instanceof Response ? await error.text() : error });
       }
     },
     [signal, submissionId]
   );
 
+  const setRuns = async (runsResponse) => {
+    const runsAnnotatedWithDurations = _.map(
+      (r) => _.merge(r, { duration: getDuration(r.state, r.submission_date, r.last_modified_timestamp, isRunInTerminalState) }),
+      runsResponse.runs
+    );
+    setRunsData(runsAnnotatedWithDurations);
+    // console.log(runsData);
```

Comment:
I think we still want to remove this commented out console.log

Reply:
I must've missed this one before, it is deleted now!